.buildkite/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:1.54
+
+# INSTALL NODE ============================================================================
+# Unfortunately can't use the friendly lts name here easily, because then it gets hard to add to the
+# path. We'll just have to update this as needed.
+ENV NODE_VERSION="v14.17.5"
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash
+ENV NVM_DIR=/root/.nvm
+RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
+RUN . "$NVM_DIR/nvm.sh" && nvm use ${NODE_VERSION}
```

Comment:
Why is this required?

Reply:
It's just the easiest way to get node installed in the docker image

Reply:
I was asking about `nvm use`, I don't think it does anything here.